using_on_other_platforms.md

```diff
@@ -0,0 +1,102 @@
+# Using APDFL and Maven on other platforms
+
+## Installing APDFL for other platforms
+
+[Maven Central](https://central.sonatype.com/artifact/com.datalogics.pdfl/pdfl) offers self-activating license managed APDFL packages for 64-bit Windows, 64-bit Intel Linux, and Mac systems
+running Apple Silicon. The samples in this repo will automatically download the APDFL package for your platform via Maven and obtain a 30 day trial license.
```

Comment:
why is this on a new line?

Reply:
I broke the lines in my editor where it made sense. I agree that's arbitrary, but my understanding is that Markdown renderers ignore line breaks in text, so putting them in prevents having to scroll sideways during editing, and doesn't affect what customers see.